uaa/src/test/java/org/cloudfoundry/identity/uaa/integration/OrchestratorZoneControllerIntegrationTests.java

```diff
@@ -143,7 +143,7 @@ public void testGetZone() {
             expectedConnectionDetails.setSubdomain(zoneName);
             expectedConnectionDetails.setUri("http://" + zoneName + ".localhost:8080/uaa");
             expectedConnectionDetails.setDashboardUri("http://localhost:8080/dashboard");
-            expectedConnectionDetails.setIssuerId(expectedConnectionDetails.getUri() + "/oauth/token");
+            expectedConnectionDetails.setIssuerId("http://" + zoneName + ".localhost:8080/uaa" + "/oauth/token");
```

Comment:
Same question as for MockMvc testing... uaaUrl and issuerUri are set to the same domain. We are not really testing that if configured differently, connection details will return different url...

Is it enough to test with the unit tests and not mock and integration?